packages/layouts/src/classes/galleryItem.js

```diff
@@ -106,21 +147,23 @@ class GalleryItem {
       cropRatio: this.cropRatio,
       isTransparent: this.isTransparent,
       offset: this.offset,
-      style: {
-        ratio: this.ratio,
-        bgColor: this.bgColor,
-        maxWidth: this.maxWidth,
-        maxHeight: this.maxHeight,
-        infoWidth: this.infoWidth,
-        infoHeight: this.infoHeight,
-        orientation: this.orientation,
-        ...this.style,
-      },
+      style,
       isDemo: this.isDemo,
       videoUrl: this.videoUrl,
       isExternalVideo: this.isExternalVideo,
+      hasSecondaryMedia: this.hasSecondaryMedia,
       ...config,
     };
+    if (this.hasSecondaryMedia) {
+      itemProps.secondaryMediaItem = {
+        ...this.secondaryMediaItem.renderProps(),
+        style: {
```

Comment:
should be specific

Reply:
Spreading the secondaryMedia props and then adding the style from the main item except the bgColor that has to be the secondarymedia item's bgColor